File: sem2/1.py
def findMean(a, n):
 
    sum = 0
    for i in range(0, n):
        sum += a[i]
 
    return float(sum/n)

def findMedian(a, n):
 
    sorted(a)
    if n % 2 != 0:
        return float(a[int(n/2)])
 
    return float((a[int((n-1)/2)] +
                  a[int(n/2)])/2.0)
 

a=[]
n = int(input("size: "))
print("elements:")
for i in range(0,n):
    num = int(input())
    a.append(num)

print("Mean =", findMean(a, n))
print("Median =", findMedian(a, n))

File: sem2/1.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int n, sum=0;
    cout << "size of array : ";
    cin >> n;
    int v[n];
    for(int i=0;i<n;i++)
        cin >> v[i];

    for(int i=0 ;i <n;i++)
        sum+=v[i];

    cout << "mean is  "<< float(sum)/float(n) << " ";

    sort(v, v+n);

    if(n%2!=0){
        cout << "median is " << v[(n-1)/2];
    }
    else cout << "median is " << v[n/2] << " " << v[n/2 + 1];

}